pkg/metrics/metrics.go

```diff
@@ -46,54 +46,62 @@ func defineMetric(name, help string, t metricType, labels ...string) MetricDefin
 }
 
 var (
-	hmapEvictionsTotal = defineMetric(
-		"hashmap_evictions_total",
-		"Number of hashmap evictions total",
+	evictionsTotal = defineMetric(
+		"evictions_total",
+		"Number of eviction events",
 		TypeCounter,
+		"source",
+		"reason",
 	)
-	userspaceNumberOfEvictionsTotal = defineMetric(
-		"userspace_number_of_evictions_total",
-		"Number of userspace evictions total",
+	evictedFlowsTotal = defineMetric(
+		"evicted_flows_total",
+		"Number of evicted flows",
 		TypeCounter,
+		"source",
+		"reason",
 	)
-	numberOfevictedFlowsTotal = defineMetric(
-		"number_of_evicted_flows_total",
-		"Number of evicted flows Total",
-		TypeCounter,
-	)
-	numberofFlowsreceivedviaRingBufferTotal = defineMetric(
-		"number_of_flows_received_via_ring_buffer_total",
-		"Number of flows received via ring buffer total",
+	evictedPktTotal = defineMetric(
+		"evicted_packets_total",
+		"Number of evicted packets",
 		TypeCounter,
+		"source",
+		"reason",
 	)
 	lookupAndDeleteMapDurationSeconds = defineMetric(
 		"lookup_and_delete_map_duration_seconds",
-		"Lookup and delete map duration seconds",
+		"Lookup and delete map duration in seconds",
 		TypeHistogram,
 	)
-	numberOfWrittenRecordsTotal = defineMetric(
-		"number_of_written_records_total",
-		"Number of written records total",
+	droppedFlows = defineMetric(
+		"dropped_flows_total",
+		"Number of dropped flows",
 		TypeCounter,
-		"exporter",
+		"source",
+		"reason",
 	)
-	exportedBatchSizeTotal = defineMetric(
-		"exported_batch_size_total",
-		"Exported batch size total",
+	bufferSize = defineMetric(
+		"buffer_size",
+		"Buffer size",
+		TypeGauge,
+		"name",
+	)
+	exportedBatchCounterTotal = defineMetric(
+		"exported_batch_total",
+		"Exported batches",
 		TypeCounter,
 		"exporter",
 	)
-	samplingRateSeconds = defineMetric(
-		"sampling_rate_seconds",
-		"Sampling rate seconds",
+	samplingRate = defineMetric(
+		"sampling_rate",
+		"Sampling rate",
```

Comment:
adding the unis was in the metrics guide line u shared with me ?

Reply:
but this one is the sampling rate (ratio) like 1:50 or 1:1 it's not measuring time